src/test_neo2owl_config.py

```diff
@@ -0,0 +1,5 @@
+from schema_test_tools import test_local
+
+test_local(path_to_schema_dir='config/prod/',
+           schema_file='neo4j2owl_config_schema.json',
+           test_dir='config/prod/')
```

Comment:
If this is run in the root directory like `python src.test_neo4j2owl_config.py`, everything is fine. But this if one would try to run it in the src directory like I did, `config/prod/` should be `../config/prod/` 
I'm not sure how do we handle this situation

Reply:
I think this is acceptable in a runner script, but happy for you to come up with something more elegant.

Reply:
>If this is run in the root directory like python src.test_neo4j2owl_config.py, everything is fine. But this if one would try to run it in the src directory like I did, config/prod/ should be ../config/prod/
I'm not sure how do we handle this situation

You can do a quick check that the subdirectories found are the expected ones. Throw an informative error. 

Either `os` or `pathlib` is useful here: https://stackoverflow.com/questions/3812849/how-to-check-whether-a-directory-is-a-sub-directory-of-another-directory